Services/UserService.cs

```diff
@@ -29,55 +30,27 @@ public UserService(IOptions<DirectoryConfig> config)
                 throw new Exception("Invalid config");
             }
 
-            var insecure = opts.Insecure;
-            var directoryServiceURL = opts.ServiceUrl;
+            var logggerFactory = new NullLoggerFactory();
```

Comment:
Maybe we should log what happens in the directory client: https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line